test/plugin/test_out_typecast.rb

```diff
@@ -28,6 +28,7 @@ def test_configure
     assert_equal('string', d.item_types['test2'])
     assert_equal('time', d.item_types['test3'])
     assert_equal('bool', d.item_types['test4'])
+    assert_equal('json', d.item_types['test5'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.